### ISSUE TITLE
mobile navigation toggle question

### ISSUE THREAD

**Issue by aberkow** [wpcampus/wpcampus-gatsby-theme#35]
I was looking at how to move the mobile navigation component fully into react instead of having some of the functionality in the `js/nav-primary.js` file. but I was a little confused by some of the functions. so I wanted to ask before moving ahead.

in the [updateMenuToggle](https://github.com/wpcampus/wpcampus-gatsby-theme/blob/master/src/js/nav-primary.js#L58-L71) method, a `js-menu-toggle` and aria attributes are added. is that because they're only relevant if JS is enabled in the browser? can they be safely moved directly to the react components? if so, I can submit a PR similar to #34.

**Comment by bamadesigner** [comment]
Yes, they're in the nav-primary.js code because they're only needed if JS is enabled.
